src/System.Console/src/System.Console.csproj

```diff
@@ -40,7 +42,14 @@
       <Link>Common\System\IO\EncodingHelper.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(TargetGroup)' != 'net46' ">
+  <!-- Windows : WinRT -->
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(TargetGroup)' == 'netcore50' ">
+    <TargetingPackReference Include="Windows" />
+    <ProjectReference Include="$(SourceDir)/mscorlib.WinRT-Facade/mscorlib.WinRT-Facade.csproj" />
+    <Compile Include="System\ConsolePal.WinRT.cs" />
+  </ItemGroup>
+  <!-- Windows : Win32 -->
+  <ItemGroup Condition=" '$(TargetsWindows)' == 'true' And '$(TargetGroup)' != 'net46' And '$(TargetGroup)' != 'netcore50'">
```

Comment:
I generally prefer positive conditions here so I would change it to `'$(TargetGroup)' == ''`